### PR TITLE
updated my profile

### PR DESCRIPTION
**Pull request by hareom284**

### CODE REVIEW

File: public/data/hareom284.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            "title": "Started Using twiiter",
+            "image": "https://github.com/hareom284.png",
+            "date": "June 2021",
+            "icon": "twitter",
+            "color": "red",
+            "description": "Creation content on twitter and share about my journey",
+            "url": "https://hareom284.me"
+        }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/hareom284.json:29]: Missing an endline.
```suggestion
}

```

File: public/data/hareom284.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/hareom284",
+            "icon": "twitter"
+        }
+    ],
+    "milestones": [
+        
+        {
+            "title": "Started Using twiiter",
```
Comment (Panquesito7) [inline public/data/hareom284.json:20]:
```suggestion
            "title": "Started Using Twitter",
```

File: public/data/hareom284.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            "image": "https://github.com/hareom284.png",
+            "date": "June 2021",
+            "icon": "twitter",
+            "color": "red",
+            "description": "Creation content on twitter and share about my journey",
```
Comment (Panquesito7) [inline public/data/hareom284.json:25]:
```suggestion
            "description": "Creation content on Twitter and sharing about my journey",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please revert the changes in the `package-lock.json` file.

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Please see the comments left above

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Please see the above comments

File: public/data/hareom284.json (hunk 2 above)
Comment (Panquesito7) [inline public/data/hareom284.json:20]:
```suggestion
            "title": "Started Using Twitter",
```

File: public/data/hareom284.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            "title": "Started Using twiiter",
+            "image": "https://github.com/hareom284.png",
+            "date": "June 2021",
+            "icon": "twitter",
+            "color": "red",
+            "description": "Create Content on Twitter and Share  my journey",
+            "url": "https://hareom284.me"
+        }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/hareom284.json:29]: Still missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Please check out the comments left above 🙂

**Review by shreyaansjain06 — CHANGES_REQUESTED** [review]
Please see suggestions above

**Comment by hareom284** [comment]
> Please see suggestions above

Please check in out I have correct that one

File: public/data/hareom284.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+            "title": "Started Using Twiiter",
+            "image": "https://github.com/hareom284.png",
+            "date": "June 2021",
+            "icon": "twitter",
+            "color": "red",
+            "description": "Create Content on Twitter and Shareing about  my journey",
+            "url": "https://hareom284.me"
+        }
+    ]
+}
```
Comment (krishguptadev) [inline public/data/hareom284.json:28]:
```suggestion
}

```

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Can you format the file and revert the package files?

File: public/data/hareom284.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        }
+    ],
+    "milestones": [
+        {
+            "title": "Started Using Twiiter",
+            "image": "https://github.com/hareom284.png",
+            "date": "June 2021",
+            "icon": "twitter",
+            "color": "red",
+            "description": "Create Content on Twitter and Shareing about  my journey",
```
Comment (ssensalo) [inline public/data/hareom284.json:24]:
```suggestion
            "description": "Create Content on Twitter and Sharing about  my journey",
```

**Review by ssensalo — COMMENTED** [review]
Please remove `package-lock.json` file also add @Panquesito7 's suggestions

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.

**Comment by Vyvy-vi** [comment]
:) Thanks for adding your profile to LinkFree!
We'll remove the `package-lock.json` file, while merging this in a staging branch

**Comment by krishguptadev** [comment]
@Vyvy-vi it would be nice if you could format the file as well, before it is merged into main

**Pull request merged**